libs-scala/ports/src/main/scala/com/digitalasset/ports/PortFiles.scala

```diff
@@ -36,9 +36,11 @@ object PortFiles {
     }
 
   private def writeUnsafe(path: Path, port: Port): Unit = {
```

Comment:
Can we get a test case for this logic?

Reply:
I think there is already one in PortFilesSpec. But yes, I'll check that there are tests, this is still a draft, I want to see all the tests that are failing now.